modules/contact/doc/content/modules/contact/index.md

```diff
@@ -270,3 +270,38 @@ The recommended PETSc options for use with `NodeFaceConstraint` based contact ar
 ## Objects, Actions, and Syntax
 
 !syntax complete groups=ContactApp level=3
+
+## Scaling effects
+
+The effects of scaling on non-linear solve convergence are shown below for
+different kinds of contact formulations. The results are based on the input files for
+[RANFS](/ranfs-and-scaling/bouncing-block-ranfs.i),
+[kinematic](/bouncing-block-kinematic.i), and
+[tangential penalty](/bouncing-block-tan-pen.i),
+running with two processes.
+
+| Scheme | Cumulative nonlinear iterations | Cumulative linear iterations |  Condition number | Scaling Factor |
+| - | - | - | - | - |
+| SMP PJFNK RANFS no scaling AMG | 61 | 310 | 6e3 | 1 |
+| SMP PJFNK RANFS residual auto-scaling AMG | 55 | 226 | 1e1 | 1.7e-2 |
+| SMP PJFNK RANFS Jacobian auto-scaling AMG | 56 | 228 | 2e1 | 3.7e-4 |
+| SMP PJFNK Kinematic no scaling AMG | 53 | 207 | 6e3 | 1 |
+| SMP PJFNK Kinematic residual auto-scaling AMG | 53 | 207 | 1e2 | 1.7e-2 |
+| SMP PJFNK Kinematic Jacobian auto-scaling AMG | 53 | 207 | 1e1 | 3.7e-4 |
+| SMP PJFNK Kinematic no scaling LU | 41 | 115 | 6e3 | 1 |
+| SMP PJFNK Kinematic residual auto-scaling LU | 41 | 115 | 1e2 | 1.7e-2 |
+| SMP PJFNK Kinematic Jacobian auto-scaling LU | 41 | 115 | 1e1 | 3.7e-4 |
+| SMP PJFNK Tangential Penalty no scaling AMG | 61 | 293 | 6e3 | 1 |
+| SMP PJFNK Tangential Penalty residual auto-scaling AMG | 61 | 293 | 1e2 | 1.7e-2 |
+| SMP PJFNK Tangential Penalty Jacobian auto-scaling AMG | 61 | 293 | 1e1 | 3.7e-4 |
+| SMP PJFNK Tangential Penalty no scaling LU | 50 | 198 | 6e3 | 1 |
+| SMP PJFNK Tangential Penalty residual auto-scaling LU | 50 | 198 | 1e2 | 1.7e-2 |
+| SMP PJFNK Tangential Penalty Jacobian auto-scaling LU | 50 | 198 | 1e1 | 3.7e-4 |
+| FD PJFNK RANFS residual auto-scaling LU | 41 | 48 | 1e1 | 1.7 e-2 |
+| FD PJFNK Kinematic residual auto-scaling LU | 56 | 80 | 1e2 | 1.7e-2 |
+
+Important takeaways:
+
+- The solve efficiency of kinematic and tangential penalty formulations is independent of scaling (within the window of this problem)
+- Because RANFS constraint residuals/Jacobians are of gap magnitude, the RANFS solve really does perform better when the internal physics is scaled to be on the same order of magnitude
+- There are some bugs with the RANFS Jacobian functions because RANFS takes more non-linears and linears than kinematic, but with a FD Jacobian RANFS takes significantly less non-linears and linears
```

Comment:
This needs to be tightened up a bit.  What do you mean by 'the window if this problem'? Change 'linears' and 'non-linears' to 'linear and nonlinear iterations'.

Reply:
> What do you mean by 'the window if this problem'?

I don't know what I mean. The whole idea behind automatic scaling was to improve the condition number of the operator and hence to improve the convergence of the linear solve (and hence to improve the convergence of the nonlinear solve). But recent weeks have shown that simply improving the condition number doesn't necessitate an improvement in the linear solve. That's what we're seeing here for kinematic and for tangential penalty and to a much more exaggerated extent in https://github.com/idaholab/moose/issues/12601#issuecomment-580574398. Though the condition number changes with scaling, the linear solve proceeds identically. So through discussions with @fdkong I've come to understand that the spectrum of eigenvalues, and how well they are clustered, is a better predictor of linear solve efficiency than just a single condition number. However, it seems that even the relationship between the eigenvalue spectrum and convergence of Krylov methods is still an open area of research.